src/GraphQl/Subscription/MercureSubscriptionIriGenerator.php

```diff
@@ -25,12 +26,12 @@
 final class MercureSubscriptionIriGenerator implements MercureSubscriptionIriGeneratorInterface
 {
     private $requestContext;
-    private $hub;
+    private $mercure;
 
-    public function __construct(RequestContext $requestContext, string $hub)
+    public function __construct(RequestContext $requestContext, Mercure $mercure)
```

Comment:
This one is a BC break but since the GraphQL system is experimental, it is allowed (but it would be better to avoid it).